src/screens/EditFlowSensorScreen.js

```diff
@@ -26,18 +27,24 @@ class EditFlowSensorScreen extends InjectedComponent<InjectedProps> {
     tabBarLabel: 'Flow Sensor',
   };
 
-  render() {
+  @computed
+  get _flowSensorLoader(): LoadObject<FlowSensor> {
```

Comment:
I assume this worked to make the warning go away?

Reply:
yes